src/plugins/ssh/index.ts

```diff
@@ -385,6 +386,9 @@ export const sshOrScp = async (
       );
     }
 
+    const maxRetries =
+      data.type === "gcloud" ? GCP_MAX_SSH_RETRIES : MAX_SSH_RETRIES;
```

Comment:
Let's please make this an exhaustive type check. Otherwise we'll forget to update this if we add another CSP.

Reply:
This is intentional: GCP is a special case with extra long wait time. I didn't mean to make this exhaustive.